services/telegram/service.go

```diff
@@ -0,0 +1,101 @@
+package telegram
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type Service struct {
+	chatId                string
+	parseMode             string
+	disableWebPagePreview bool
+	disableNotification   bool
+	url                   string
+	global                bool
+	stateChangesOnly      bool
+	logger                *log.Logger
+}
+
+func NewService(c Config, l *log.Logger) *Service {
+	return &Service{
+		chatId:                c.ChatId,
+		parseMode:             c.ParseMode,
+		disableWebPagePreview: c.DisableWebPagePreview,
+		disableNotification:   c.DisableNotification,
+		url:                   c.URL + c.Token + "/sendMessage",
+		global:                c.Global,
+		stateChangesOnly:      c.StateChangesOnly,
+		logger:                l,
+	}
+}
+
+func (s *Service) Open() error {
+	return nil
+}
+
+func (s *Service) Close() error {
+	return nil
+}
+
+func (s *Service) Global() bool {
+	return s.global
+}
+func (s *Service) StateChangesOnly() bool {
+	return s.stateChangesOnly
+}
+
+func (s *Service) Alert(chatId, parseMode, message string, disableWebPagePreview, disableNotification bool) error {
+	if chatId == "" {
+		chatId = s.chatId
+	}
+
+	if parseMode == "" {
+		parseMode = s.parseMode
+	}
+
+	postData := make(map[string]interface{})
+	postData["chat_id"] = chatId
+	postData["text"] = message
+	postData["parse_mode"] = parseMode
+	postData["disable_web_page_preview"] = disableWebPagePreview
```

Comment:
I'm a bit confused how to check it, cause they can't be nil, and if they are not passed in tick, they are false. Maybe this is OK?